requirements.txt

```diff
@@ -1,2 +1,4 @@
 twisted
-pika
\ No newline at end of file
+pika
+tabulate
```

Comment:
Since this seems to be only required for tests, maybe we could have a ``requirements-test.txt`` file